!exec function b(n){try{return b(n+1)}catch(e){}};b(0)